.gitignore

```diff
@@ -65,3 +65,5 @@ migrations/cockroach/migrate_cloud.go
 !/.artifacts/zitadel
 /zitadel
 
+go.work
+go.work.sum
```

Comment:
is this needed?

Reply:
I you use the workspace feature of go, e.g. to replace some imports for debugging purposes, these files are created. Thats why I added them to gitignore